test/built-ins/Promise/try/throws.js

```diff
@@ -6,12 +6,15 @@ description: Promise.try returns a Promise that rejects when the function throws
 esid: sec-promise.try
 features: [promise-try]
 flags: [async]
+includes: [asyncHelpers.js]
 ---*/
 
-assert.throwsAsync(
-  Test262Error,
-  function () {
-    Promise.try(function () { throw new Test262Error(); })
-  },
-  "error thrown from callback must become a rejection"
-);
+asyncTest(async function() {
+  await assert.throwsAsync(
+    Test262Error,
+    function () {
+      Promise.try(function () { throw new Test262Error(); })
+    },
+    "error thrown from callback must become a rejection"
+  );
+});
```

Comment:
it was very intentional to not use async/await syntax here; we try to avoid using syntax that isn't necessary for the test. I'll put up a PR that fixes it.

Reply:
#4087